src/pages/Cart/Cart.js

```diff
@@ -1,9 +1,77 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+
+import CartTotal from './components/CartTotal/CartTotal';
+import ContentsContainer from './components/ContentsContainer/ContentsContainer';
+
+import './Cart.scss';
 
 const Cart = () => {
+  console.log('render!');
+  const [cartItems, setCartItems] = useState([]);
+  const [checkedItems, setCheckedItems] = useState(new Set());
```

Comment:
```suggestion
  const [checkedItems, setCheckedItems] = useState([]);
```

초기값으로 빈 배열로 설정해도 되겠네요!